frontend/src/test/Background.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { shallow } from "enzyme";
+import Background from "../components/eMIB/Background";
+
+it("renders without crashing", () => {
```

Comment:
No need to include the "renders without crashing" tests anymore - they'll just take up time and will be checked anyways now that we have other tests.